src/fields/BooleanField.cpp

```diff
@@ -1,7 +1,7 @@
 #include "BooleanField.h"
 
-BooleanField::BooleanField(string name, bool defaultValue, bool primary, bool unique, bool nullable): 
-    Field(name, BOOLEAN, primary, unique, nullable) { 
+BooleanField::BooleanField(string fieldName, bool defaultValue, bool primaryKey, bool unique, bool nullable): 

```

Comment:
It is a nice naming convention. Plus it's still fairly early days, so I don't mind putting in a bit of extra work for stuff like this (while it's relatively easy to do so without breaking all the things).